dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops

```diff
@@ -870,6 +870,10 @@ ERROR_INVALID_PRODUCT_EDITION_FEATURE_UTILITY=CWWKF1410E: The {0} feature cannot
 ERROR_INVALID_PRODUCT_EDITION_FEATURE_UTILITY.explanation=The feature is not applicable to the edition of the installed product.
 ERROR_INVALID_PRODUCT_EDITION_FEATURE_UTILITY.useraction=Install features that are supported by the product edition.
 
+ERROR_BETA_EDITION_NOT_SUPPORTED=CWWKF1411E: Installing or updating features is not supported for Beta releases.
```

Comment:
"Beta" doesn't need to be capitalized (unless it is throughout the file or something like that).

Reply:
change made, thanks!